include/ck/tensor_operation/gpu/grid/gridwise_set_buffer_value.hpp

```diff
@@ -3,6 +3,7 @@
 
 #pragma once
 
+#include "ck/tensor_operation/gpu/element/element_wise_operation.hpp"
```

Comment:
Why you add the new header without any code change?

Reply:
To refer to the `ck::elemen_twise::PassThrough` used by the ThreadwiseTensorTransfer_v1r3

Reply:
`gridwise_set_buffer_value.hpp` is not used by batchnorm after we take use of `GetMaxCountPerThread`, but is still used by multiblock reduction kernel